backends/magma/ceed-magma.c

```diff
@@ -36,6 +45,23 @@ static int CeedInit_Magma(const char *resource, Ceed ceed) {
     return CeedError(ceed, 1, "error in magma_init(): %d\n", ierr);
   // LCOV_EXCL_STOP
 
+  Ceed_Magma *data;
+  ierr = CeedCalloc(sizeof(Ceed_Magma), &data); CeedChk(ierr);
+  ierr = CeedSetData(ceed, (void *)&data); CeedChk(ierr);
+
+  // kernel selection
+  data->basis_kernel_mode = MAGMA_KERNEL_DIM_SPECIFIC;
+  //data->basis_kernel_mode = MAGMA_KERNEL_DIM_GENERIC;
```

Comment:
Dead code should be removed.

Reply:
Done in 195ca8ef85eeb68ebbdc2c75c6f28c876ee1d0a1